composer.json

```diff
@@ -24,29 +24,30 @@
         "doctrine/persistence": "^1.3 || ^2.1",
         "egulias/email-validator": "^2.1.10",
         "paragonie/random_compat": "^1 || ^2 || ^9",
-        "symfony/config": "^4.4 || ^5.0",
-        "symfony/console": "^4.4 || ^5.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0",
-        "symfony/event-dispatcher": "^4.4 || ^5.0",
+        "symfony/config": "^5.3 || ^6.0",
```

Comment:
Please remove the 6.0 versions as long as there is no stable release and the CI does not cover this version.

Reply:
That makes sense.